### ISSUE TITLE
"partial" VR support

### ISSUE THREAD

**Issue by evilbuddy** [Ryujinx/Ryujinx#3381]
Adding a "partial" support for VR would be nice.
Here is what I understand for partial VR support:
 - view on a big screen (like the SteamVR desktop tool, or Bigscreen)
 - support for controllers (buttons, motion controls, etc...)
 
 Compared to a "full" VR support, we are not "in" the game, but we have a partial support, mainly for motion controls with VR controllers.

This could be done using Oculus Desktop or SteamVR API.

And this would be cool, especially for motions controls.

**Comment by mstreurman** [comment]
Or even full support as there are a few games that have LaboVR

**Comment by evilbuddy** [comment]
Yeah, it would be even better for supported games. But I think that the partial support would be a first step before full support
